### PR TITLE
Merge `alpha-testing-v-19` into `develop`

### PR DESCRIPTION
**Pull request by jamil314**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

**Review by cibelius — APPROVED** [review]

File: src/client-config.prod.js [hunk 1]
```diff
@@ -36,6 +36,5 @@ window.config = {
   SENTRY: sentry,
   LEADERBOARDS_DASHBOARD_URL: `${scheme}//metabase.${hostname}/public/dashboard/acae0527-74be-4804-a3ee-f8b3c9c8784c#bordered=false&titled=false&refresh=300`,
   REGISTRATIONS_DASHBOARD_URL: `${scheme}//metabase.${hostname}/public/dashboard/fec78656-e4f9-4b51-b540-0fed81dbd821#bordered=false&titled=false&refresh=300`,
-  STATISTICS_DASHBOARD_URL: `${scheme}//metabase.${hostname}/public/dashboard/a17e9bc0-15a2-4bd1-92fa-ab0f346227ca#bordered=false&titled=false&refresh=300`,
-  FEATURES: {}
```
Comment (Zangetsu101) [inline src/client-config.prod.js:40]: We need to keep the `FEATURES` property otherwise core would error

**Review by Zangetsu101 — APPROVED** [review]
